posthog/api/unsubscribe.py

```diff
@@ -0,0 +1,17 @@
+import jwt
+from django.http import HttpRequest, JsonResponse
+
+from posthog.models.subscription import unsubscribe_using_token
+
+
+def unsubscribe(request: HttpRequest):
```

Comment:
should this be in`ee` too?

subscribe is `ee` but unsubscribe is `posthog` 🤔

Reply:
That's a good point. I guess my thinking was, if someone subscribes whilst on the Scale plan but downgrades later they should still be able to unsubscribe?

Reply:
🤯